_maps/RandomRuins/IceRuins/icemoon_underground_abandoned_plasma_facility.dmm

```diff
@@ -358,7 +358,7 @@
 /turf/open/lava/plasma/ice_moon,
 /area/icemoon/underground/explored)
 "gd" = (
-/mob/living/simple_animal/hostile/asteroid/wolf,
+/mob/living/basic/mining/wolf,
```

Comment:
please make an updatepaths for this so downstreams can automatically benefit from this: https://github.com/tgstation/tgstation/tree/master/tools/UpdatePaths